open-api/rest-catalog-open-api.yaml

```diff
@@ -2511,7 +2511,7 @@ components:
         format-version:
           type: integer
           minimum: 1
-          maximum: 2
+          maximum: 4
```

Comment:
Do we also need to update the TCK tests for v3 and v4?

Reply:
Good point, I think so yeah. I will update the PR.